build.sbt

```diff
@@ -1,24 +1,37 @@
 lazy val root = (project in file("."))
   .settings(noPublishSettings)
   .aggregate(coreJVM, coreJS)
+  .aggregate(testsJVM, testsJS)
   .aggregate(examplesJVM, examplesJS)
   .aggregate(bench)
 
 lazy val core = module("core", hideFolder = true)
   .settings(scalaMacroDependencies)
   .settings(yax(file("modules/core/src/main/scala"), Compile,
     yaxScala = true))
-  .settings(yax(file("modules/core/src/test/scala"), Test,
-    yaxPlatform = true))
   .crossDepSettings(
     %%("cats-core", V.cats),
```

Comment:
Unrelated, but you could drop the dependency on `cats-core` as `cats-free` gives you that as well.